docs/multi-agent-applications.md

```diff
@@ -330,11 +330,6 @@ graph TB
   seat_preference_agent --> END
 ```
 
-## PydanticAI Graphs
-
-!!! example "Work in progress"
-    This is a work in progress and not yet documented, see [#528](https://github.com/pydantic/pydantic-ai/issues/528) and [#539](https://github.com/pydantic/pydantic-ai/issues/539)
-
```

Comment:
I feel it's worth keeping this section with just a link to the other page, so you don't have to scroll back up to the original numbered list to get the link if you read this far in the docs and are curious about the graph stuff.